bedrock/firefox/templates/firefox/private-browsing.html

```diff
@@ -0,0 +1,136 @@
+{# This Source Code Form is subject to the terms of the Mozilla Public
+ # License, v. 2.0. If a copy of the MPL was not distributed with this
+ # file, You can obtain one at http://mozilla.org/MPL/2.0/. -#}
+
+{% from "macros.html" import fxfamilynav with context %}
+
+{% extends "/firefox/base-resp.html" %}
+
+{% block page_title %}{{ _('Keep your Firefox in sync') }}{% endblock %}
+{% block body_id %}firefox-private-browsing{% endblock %}
+{% block body_class %}{{ super() }} mozid{% endblock %}
+
+{% block page_css %}
+  {% stylesheet 'firefox_private_browsing' %}
+{% endblock %}
+
+{% block site_header %}{% endblock %}
+
+{% block site_header_unwrapped %}
+  {{ fxfamilynav('desktop', 'private-browsing') }}
+{% endblock %}
+
+{% block content %}
+
+<main role="main">
+
+  <section class="intro">
+    <div class="container">
+      <div class="header">
+        <h1>
+          {{ _('More protection.') }}
+          <br>
+          {{ _('The most privacy.') }}
+          <br>
+          {{ _('Only from Firefox.') }}
```

Comment:
@flodolo To be clear here, it should change in this case because we already have that string in the landing page, but usually this would be a better way to wrap the strings, right?